core/src/com/etheller/warsmash/util/Quadtree.java

```diff
@@ -68,9 +68,7 @@ public boolean intersect(final Rectangle bounds, final QuadtreeIntersector<T> in
 				}
 			}
 			if (this.southeast.bounds.overlaps(bounds)) {
-				if (this.southeast.intersect(bounds, intersector)) {
-					return true;
-				}
+				return this.southeast.intersect(bounds, intersector);
```

Comment:
Isn't this an exceedingly pedantic automated change the interrupts the original style of the code where there were four identical branches that could be easily observed by a developer to be the same (see image for how this code looked on my PC before the change):
![image](https://user-images.githubusercontent.com/5471783/165432423-bcb14bb7-e5d1-4242-8d38-4baf1c50d730.png)

Certainly it is correct either way, but if we were ever going to modify this Quadtree to make an Octtree for example, and copy this to have eight branches instead of four, having one of the branches be needlessly different in order to pedantically avoid "if {x} return" syntax seems pointless from my perspective. Curious to hear your opinion or if there's a good reason in favor of changing it that I'm just not seeing.

Reply:
I agree with you, having one of the branches be needlessly different in order to pedantically avoid "if {x} return" syntax is pointless. Please check 60abd007b901f47ca1d9b8cf424f7608c6e7b8bc, where are refactored Quadtree.